u2-routing-controllers/features/tempController.js

```diff
@@ -0,0 +1,20 @@
+const getTemp = (req, res) => {
+  const u1 = req.params.u1.toUpperCase()
+  const u2 = req.params.u2.toUpperCase()
+  const t1 = req.params.t1
```

Comment:
คุณควรแยกส่วนนี้เป็นทศนิยมก่อนที่จะใช้ในการคำนวณ `t2`:
You should parse this to a float before using it in the calculation of `t2`:
```js
const t1 = parseFloat(req.params.t1) || 0.0
```